src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1502,6 +1502,51 @@ public static IEnumerable<string> ParseConditionalCompilationSymbols(string valu
             return defines.AsEnumerable();
         }
 
+        public static bool TryParseLanguageVersion(string value, out LanguageVersion version)
```

Comment:
Yes. After chat with Omar, the plan if for my PR to go in first. 
I updated my PR to remove the IDE parsing function and other improvements that Omar was going to do.